x-pack/test/functional/apps/infra/node_details.ts

```diff
@@ -233,6 +233,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         describe('Processes Tab', () => {
           before(async () => {
             await pageObjects.assetDetails.clickProcessesTab();
+            await pageObjects.header.waitUntilLoadingHasFinished();
```

Comment:
Do you think we need to add the same thing for other tabs as well?

Reply:
We have it when initially loading the page - in theory, repeating this is not needed for most cases there is also a timeout so it shouldn't reach it. Here we are checking the content of an element that is still not available which results in having an empty string as value - I guess because we use `getVisibleText` ( the other tabs we check the elements directly and here we try to get the content of an element which is still not loaded)
I am not against having this in the other tabs but I don't see any failed tests in the other scenarios, maybe I am missing something or it's not an issue there 🤔

Reply:
Ah, got it, good point about `getVisibleText`, that's probably the difference with other tabs and we don't need to add wait for them 🙌